tsconfig.json

```diff
@@ -0,0 +1,73 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Basic Options */
+    // "incremental": true,                   /* Enable incremental compilation */
+    "target": "es2020" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */,
```

Comment:
I'd like to target older browsers, so I think this should be ES5.

Reply:
This is what `tsc` put out, we then run it through babel to get the `dist/**bundled.js` version. That bundled version is used by webpack for the `dist/**min.js` version. I'm not sure we need all these outputs but I did it to preserve what was already output. I kept this as `es2020` so a use could copy it directly into their project if need be.

Reply:
that's brilliant!  thank you for the explanation